tensorflow/python/ops/script_ops_test.py

```diff
@@ -37,6 +39,47 @@ def plus(a, b):
     expect_result = constant_op.constant(3, dtypes.int32)
     self.assertAllEqual(actual_result, expect_result)
 
+  def test_stateless_flag(self):
+    call_count = 0
+
+    def plus(a, b):
+      global call_count
+      call_count += 1
+      return a + b
+
+    @def_function.function
+    def tensor_plus_stateful(a, b):
+      return numpy_function(plus, [a, b], dtypes.int32, stateful=True)
+
+    @def_function.function
+    def tensor_plus_stateless(a, b):
+      return numpy_function(plus, [a, b], dtypes.int32, stateful=False)
+
+    @def_function.function(autograph=False)
```

Comment:
No need for this override, I think (same below).

Reply:
Hm, true, if the nodes generated by the `nupmy_function` with the same Python function are considered the same

Reply:
Sounds good, that's a func change as well.

But I was referring to the `autograph=False` override. The test should run with just `@def_function.function` as well.

Reply:
:) good, done that as well